### PR TITLE
Adding Numaflow to Streaming & Messaging

### PR DESCRIPTION
**Pull request by syayi**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [X] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [X] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [X] Have you picked the single best (existing) category for your project?
* [X] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [X] Have you added your SVG to `hosted_logos` and referenced it there?
* [X] Does your logo clearly state the name of the project/product and follow the other [guidelines](https://github.com/cncf/landscape#new-entries)?
* [X] Does your project/product name match the text on the logo?
* [X] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?

### CODE REVIEW

File: landscape.yml [hunk 1]
```diff
@@ -5710,6 +5710,11 @@ landscape:
                   type: security
                   url: https://github.com/nats-io/nats-general/blob/master/reports/Cure53_NATS_Audit.pdf
                   vendor: Cure53
+          - item:
+            name: Numaflow
+            description: K8s native real-time data and stream processing engine
+            repo_url: https://github.com/numaproj/numaflow
+            logo: numaproj-logo.svg
```
Comment (castrojo) [inline landscape.yml:5717]:
```suggestion
          - item:
            name: Numaflow
            description: K8s native real-time data and stream processing engine
            homepage_url: https://numaflow.numaproj.io/
            repo_url: https://github.com/numaproj/numaflow
            logo: numaproj-logo.svg
```

Comment (castrojo) [reply landscape.yml:5717]: You'll need at least a homepage_url

**Review by castrojo — CHANGES_REQUESTED** [review]

**Comment by syayi** [comment]
@castrojo thanks for the review. Added the requested changes

**Review by carasen12 — APPROVED** [review]

**Comment by castrojo** [comment]
Alright all set, thanks!

**Review by castrojo — APPROVED** [review]

**Pull request merged**